homeassistant/components/izone/climate.py

```diff
@@ -0,0 +1,468 @@
+"""
+Support for the iZone HVAC.
+
+For more details about this platform, please refer to the documentation at
+https://github.com/Swamp-Ig/izone_custom_component
```

Comment:
If this gets an official integration, this is an outdated link.

Also, we don't use these comment headers anymore.